java/cdk/src/main/java/sleeper/cdk/stack/TableStack.java

```diff
@@ -102,6 +102,7 @@ public TableStack(
                 .build();
 
         dynamoDBStateStoreStack.grantReadWriteActiveFileMetadata(sleeperTableProvider.getOnEventHandler());
+        dynamoDBStateStoreStack.grantReadWriteReadyForGCFileMetadata(sleeperTableProvider.getOnEventHandler());
```

Comment:
This was necessary because in PR #1362 we added a check for whether any files exist in the table before we reinitialise it. We've now made it just check against the active files, and leave GC files alone, so this permission should not be needed.